elasticdl/python/tests/master_test.py

```diff
@@ -65,3 +65,34 @@ def test_master_run_and_stop(self):
             exit_code = master.run()
             master.stop()
             self.assertEqual(exit_code, 0)
+
+    def test_master_validate(self):
+        with tempfile.TemporaryDirectory() as temp_dir_name:
+            create_recordio_file(
+                self._num_records,
+                DatasetName.TEST_MODULE,
+                1,
+                temp_dir=temp_dir_name,
+            )
+            self.arguments["training_data"] = temp_dir_name
+            self.arguments["task_fault_tolerance"] = "False"
+            args = self._get_args()
+            args = parse_master_args(args)
+            master = Master(args)
+            with self.assertRaises(Exception):
+                master.validate()
+
+            self.arguments["need_elasticdl_job_service"] = "False"
+            args = self._get_args()
+            args = parse_master_args(args)
+            master = Master(args)
+            master.validate()
+
+            self.arguments["task_fault_tolerance"] = "False"
+            self.arguments["task_fault_tolerance"] = "True"
```

Comment:
Why do wen set `task_fault_tolerance` twice?

Reply:
Copy&Paste bug. Need to delete the 1st one.